core/node/libp2p/rcmgr_defaults.go

```diff
@@ -184,7 +176,7 @@ Run 'ipfs swarm limit all' to see the resulting limits.
 	// Whatever limits libp2p has specifically tuned for its protocols/services we'll apply.
 	libp2p.SetDefaultServiceLimits(&scalingLimitConfig)
 
-	defaultLimitConfig := scalingLimitConfig.Scale(int64(maxMemory), int(numFD))
+	defaultLimitConfig := scalingLimitConfig.Scale(int64(maxMemory), int(maxFD))
```

Comment:
if we do the int64 and int conversions above, they aren't needed here right?

Reply:
changed.